### ISSUE TITLE
Pass Aria, Role, and Tabindex values down to Anchor tag

### ISSUE THREAD

**Issue by perrygovier** [ionic-team/stencil-router#37]
For SEO and accessibility, these tags should be on the `<a>` tags if we set them as props on the `<stencil-route-link>`.

Note, we don't want to have those attributes on the route link _and_ the anchor tag, so we might want to use prefixed attributes on the route link. 

Something like: 
```html
<stencil-route-link exact url={url} anchor-title={myText} anchor-role="menuitem">
  {myText}
</stencil-route-link>
```

becomes: 

```html
<stencil-route-link class="hydrated">
  <a title="The Value of  myText" role="menuitem"  href="/the/value/of/url">
    The Value of  myText
  </a>
</stencil-route-link>
```

**Comment by adamdbradley** [comment]
Attributes that get moved down to the `<a>` could get removed from the host.
